lib/pbench/test/unit/server/query_apis/test_namespace_and_rows.py

```diff
@@ -29,7 +29,9 @@ def _setup(self, client):
             index_from_metadata="result-data-sample",
         )
 
-    def test_with_no_index_document(self, client, server_config):
+    def test_with_no_index_document(
+        self, client, server_config, pbench_token, attach_dataset
+    ):
         """
         Check the Namespace API when no index name is provided
         """
```

Comment:
Because it was previously failing before we got to the subclass authorization; now the automatic DATASET authorization triggers in `_dispatch` before the intended failure!